package.json

```diff
@@ -58,7 +58,8 @@
     "@storybook/addons": "6.0.28",
     "@storybook/react": "6.0.28",
     "@testing-library/jest-dom": "5.8.0",
-    "@testing-library/react": "10.0.4",
+    "@testing-library/react": "11.1.2",
```

Comment:
I need the latest version of @testing-library/react in order for @testing-library/react-hooks to work properly